var/spack/repos/builtin/packages/gromacs/package.py

```diff
@@ -58,12 +58,21 @@ class Gromacs(CMakePackage):
                     'AVX2_128', 'AVX2_256', 'AVX_512', 'AVX_512_KNL',
                     'IBM_QPX', 'Sparc64_HPC_ACE', 'IBM_VMX', 'IBM_VSX',
                     'ARM_NEON', 'ARM_NEON_ASIMD'))
-    variant('rdtscp', default=True, description='Enable RDTSCP instruction usage')
+    variant('rdtscp', default=True,
+            description='Enable RDTSCP instruction usage')
+    variant('subcounters', default=False,
+            description='Enable wallcycle subcounters')
+    variant('own_fftw', default=True, description='Build own FFTW')
+    variant('omp_max_threads', default='auto',
```

Comment:
Nothing wrong with this variant per se, but isn't this (or shouldn't it be) configurable at runtime?  It seems weird that this is a build option on GROMACS.

Reply:
It seems `GMX_OPENMP_MAX_THREADS` is only use to set the max bitmask: https://github.com/gromacs/gromacs/blob/master/src/gromacs/utility/bitmask.h#L59